parkhangs-webapp/src/components/CommentForm.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { addParkComment, addEventComment } from 'features/comments/commentSlice'
+import moment from 'moment'
+import './CommentForm.css'
+
+class CommentForm extends React.Component {
+
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            comments: this.comment()
+        }
+    }
+
+    comment = () => {
+        return this.props.comment || ''
+    }
+
+    handleUpdateComment = (event) => {
+        this.setState({
+            comment: event.target.value
+        })
+    }
+
+    handleAddComment = (comment) => {
+        event.preventDefault()
+
+        const commentText =  this.state.comment || this.comment()
+
+        if ("parkId" in this.props) {
```

Comment:
indents are all wonky in here haha hard to figure out the if else if else structure